modules/steam.nix

```diff
@@ -71,8 +71,6 @@ let
       # TODO[Jovian]: verify assertion
       --xwayland-count 2
 
-      -w 1280 -h 800
```

Comment:
I'm not 100% positive, but I believe this ends-up being useful to force "800p" rendering even when outputing to e.g. a 1080p or better display. The gamescope'd content uses the lower resolution, and upscaling (possibly FSR!) handles the rest, if I got things right.

Reply:
Yeah but still I don't it should be the default. Currently it causes behavior difference between the DRM and nested modes, where the latter is blurry on an external display (`-w` and `-h` only affect the nested mode). I assume it was added to the vendor script to help developers debug locally with the correct resolution and aspect ratio.

Reply:
Scratch that, I was mistaken. `-w` and `-h` define the default resolution of the application, regardless of nested or embedded mode. It can be changed by the application during runtime (in Steam, it can be configured per-game in the Properties screen). Let's keep the upstream behavior.